src/translations/en.json

```diff
@@ -831,7 +831,7 @@
         },
         "core": {
           "caption": "General",
-          "description": "Change your general Home Assistant configuration",
+          "description": "Unit system, timezone, ...",
```

Comment:
I don't like the `...`

Reply:
Then perhaps: "Unit system, location, time zone & other parameters"

Reply:
Better